extensions/wikia/CreateNewWiki/tasks/ImportStarterData.php

```diff
@@ -62,10 +69,11 @@ public function executeShell() {
 		global $IP;
 
 		$cmd = sprintf(
-			"SERVER_ID=%d %s %s/maintenance/importStarter.php",
+			"SERVER_ID=%d %s %s/maintenance/importStarter.php --vertical=%d",
 			$this->taskContext->getCityId(),
 			$this->phpBin,
-			"{$IP}/extensions/wikia/CreateNewWiki"
+			"{$IP}/extensions/wikia/CreateNewWiki",
+			$this->taskContext->getVertical()
```

Comment:
We can't. Vertical is set in [ConfigureCategories task](https://github.com/Wikia/app/blob/088d6da07d032206dd2405135e0156d8373bbb04/extensions/wikia/CreateNewWiki/tasks/ConfigureCategories.php#L55) which is run _after_ we import the starter, see [the list of tasks](https://github.com/Wikia/app/blob/ddbe1f889d91facdcb597250c6b6c01d801c58bf/extensions/wikia/CreateNewWiki/tasks/TaskRunner.php#L29-L31).